content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -10,17 +10,27 @@ weight: 10
 
 {{% capture overview %}}
 
-ReplicaSet is the next-generation Replication Controller. The only difference
+ReplicaSet is a type of Replication Controller. The only difference
```

Comment:
ReplicaSet is a replacement for the old ReplicationController. So it's not really a type of replication controller. We can probably remove all mention of ReplicationController from this topic, except for one statement that mentions ReplicationController as a thing of the past.

Reply:
done